group/views.py

```diff
@@ -38,10 +41,7 @@ def get_group(group_id):
         group = Group.objects.get(id = group_id)
     except Group.DoesNotExist:
         logger.warning('Group: Can not edit group %s! Group is not exist!' % group_id)
```

Comment:
Is the type of group_id in line43 correct?
If the type is string the and code in line41 should be
```
group = Group.objects.get(id = long(group_id))
```